Shared/Tests/Get-IISModules.Tests.ps1

```diff
@@ -0,0 +1,157 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidOverwritingBuiltInCmdlets', '', Justification = 'Pester testing file')]
+[CmdletBinding()]
+param()
+BeforeAll {
+    $Script:parentPath = (Split-Path -Parent $PSScriptRoot)
+    . $Script:parentPath\IISFunctions\Get-IISModules.ps1
+
+    function LoadApplicationHostConfig {
```

Comment:
i believe this should be a `mock` not a `function`

Reply:
This one is needs to be a (helper) function to properly load the applicationHost.config for performing the unit tests.